transformations/dev-prod-mode/dev-prod-mode.md

```diff
@@ -275,7 +275,55 @@ in the branch where they took place.*
 
 Now let’s do the same thing with tables to see how table storage behaves in Development/Production Mode. 
 
-*To be continued.*
+## Working with tables
 
+### Prepare the production configurations 
+
+Switch back to production. 
+
+Our business case is now to prepare a pipeline that pulls data about bitcoin prices and store it in storage. Then we create a transformation that stores only the last 5 values in the out stage. So let's prepare the production configurations.  
+
+First we need to pull the bitcoin data. To simplify this, we will download a [prepared csv file](/transformations/dev-prod-mode/bitcoin_price.csv) using HTTP extractor. 
+
+#### Set up http extractor in production
+
+Create new HTTP extractor configuration, fill in `Base URL` to `https://help.keboola.com`. Then add new table to the extractor, named `bitcoin_price` and fill the `Path` to `/transformations/dev-prod-mode/bitcoin_price.csv`. `Table Name` should be `bitcoin_price`. 
+
+Run the extractor and verify that a new table `in.c-keboola-ex-http-682373219.bitcoin_price` was created. *Note, that the number in your bucket name will be different.* 
+
+#### Prepare a transformation in production
+
+First create a new Snowflake transformation, named `Bitcoin`. 
+
+In the input mapping fill in the `bitcoin_price` table that you created using HTTP extractor. In output mapping add `top5` table that will be created in the transformation. 
```

Comment:
celkově bych použil toto https://help.keboola.com/tutorial/manipulate/, protože se už je to jednou vyřešený :)

Reply:
Ugh, to je škoda, že jsem to nevěděl dřív. Mezitím mám zas o kus víc hotovo :'(

Reply:
Už jsem moc daleko, tak pokračuju v tom svém usecasu.